keyboards/keychron/q1/rev_0100/rules.mk

```diff
@@ -19,3 +19,4 @@ AUDIO_ENABLE = no           # Audio output
 DIP_SWITCH_ENABLE = yes
 RGB_MATRIX_ENABLE = yes
 RGB_MATRIX_DRIVER = IS31FL3733
+LTO_ENABLE = yes
```

Comment:
Any changes should go under the `vial` keymap

Reply:
Made the necessary changes for LTO_ENABLE rule relocation.